### PR TITLE
Increase base Lighthouse score of default Create Next App template

### PR DESCRIPTION
**Pull request by garmeeh**
Very small update to increase the base lighthouse score of the default template for Create Next App.

Before:
create-next-app-before

After:
create-next-app-after

To get a full 💯 sweep, `<html>` would need a lang attribute set. I think adding a custom document or i18n to the default template would probably be just a bit too much for such a simple template.

## Documentation / Examples

- [x] Make sure the linting passes

### CODE REVIEW

File: packages/create-next-app/templates/default/pages/index.js [hunk 1]
```diff
@@ -57,7 +58,13 @@ export default function Home() {
           rel="noopener noreferrer"
         >
           Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
+          <img
```
Comment (timneutkens) [inline packages/create-next-app/templates/default/pages/index.js:61]: This can use next/image

**Comment by timneutkens** [comment]
How did you run these numbers? This change shouldn't have that type of effect, sounds like it was ran on a local machine 🤔

File: packages/create-next-app/templates/default/pages/index.js (hunk 1 above)
Comment (garmeeh) [reply packages/create-next-app/templates/default/pages/index.js:61]: I was thinking of adding next/image here but wasn't sure for the purpose of the default template did you want to include it? I can update to use if so.

**Comment by garmeeh** [comment]
@timneutkens yeah these were just run locally that is why metrics are shifting. (sorry should have cut them out of the screenshot) Main figures effected by these changes are SEO and Accessibility.

**Review by timneutkens — APPROVED** [review]

File: packages/create-next-app/templates/default/pages/index.js (hunk 1 above)
Comment (timneutkens) [reply packages/create-next-app/templates/default/pages/index.js:61]: Yeah feel free to add it in a separate PR 👍

**Pull request merged**